tools/operator/testclient/operations/channelConfig.go

```diff
@@ -123,23 +137,34 @@ func (c ChannelUIObject) createChannelObjectIfChanPrefix(channel inputStructs.Ch
 	return channelUIObjects
 }
 
+func (c ChannelUIObject) channelConfig(action, channelName string, args []string, wg *sync.WaitGroup) error {
+	defer wg.Done()
+	_, err := networkclient.ExecuteCommand("node", args, true)
+	if err != nil {
+		logger.ERROR(fmt.Sprintf("Failed to perform %s action on %s channel: %v", action, channelName, err))
+		os.Exit(1)
+	}
+	return nil
+}
+
 //doChannelAction -- To perform channel operations including create, anchorpeer update and join channel
 func (c ChannelUIObject) doChannelAction(channelUIObjects []ChannelUIObject) error {
 
 	var err error
 	var jsonObject []byte
+	var wg sync.WaitGroup
 	pteMainPath := paths.PTEPath()
-	for i := 0; i < len(channelUIObjects); i++ {
-		jsonObject, err = json.Marshal(channelUIObjects[i])
+	var args []string
+	for i, channelObject := range channelUIObjects {
+		jsonObject, err = json.Marshal(channelObject)
 		if err != nil {
 			return err
 		}
 		startTime := fmt.Sprintf("%s", time.Now())
```

Comment:
Time is a stringer, you can call `time.Now().String()` and then put that directly in the `[]string` slice directly